sdks/python/apache_beam/runners/portability/stager.py

```diff
@@ -136,6 +137,8 @@ def create_job_resources(options,  # type: PipelineOptions
             only for testing.
           populate_requirements_cache: Callable for populating the requirements
             cache. Used only for testing.
+          skip_boot_dependencies: Skip apache beam sdk, requirements, extra
+            packages, workflow tarball installs by sdk boot program.
```

Comment:
This seems confusing since stager and boot program in SDK containers are different entities.

How about `skip_prestaged_dependencies: Skip staging dependencies that can be added into SDK containers during prebuilding`.

Reply:
sounds much better, thanks.